docs/learn/learn-kusama-inflation.md

```diff
@@ -0,0 +1,99 @@
+---
+id: learn-kusama-inflation
+title: Kusama Inflation Model
+sidebar_label: KSM Inflation
+description: Explanation of KSM's Inflation Model.
+keywords: [token, KSM, inflation]
+slug: ../learn-kusama-inflation
+---
+
+import MessageBox from "../../components/MessageBox"; import "../../components/MessageBox.css";
+
+<MessageBox message="[Learn about Polkadot's native token DOT inflation](./learn-inflation)." />
+
+KSM is an inflationary token. Inflation is
+[set to be 10% annually](https://github.com/paritytech/polkadot-sdk/blob/756ccc35e93d1a78e3c71a0e67ae4da5f1d09f69/runtime/kusama/src/lib.rs#L535).
```

Comment:
This is a super old commit! the actual 10% is stored in the parameters pallet instance in the Kusama runtime.